packages/web-components/src/components/va-featured-content/va-featured-content.tsx

```diff
@@ -7,18 +7,53 @@ import { Component, Host, h } from '@stencil/core';
  */
 @Component({
   tag: 'va-featured-content',
-  styleUrl: 'va-featured-content.css',
+  styleUrl: 'va-featured-content.scss',
   shadow: true,
 })
 export class VaFeaturedContent {
+  /**
+   * Whether or not the component will use USWDS v3 styling.
+   */
+  @Prop() uswds?: boolean = false;
+
+  @Element() el: HTMLElement;
+
+  componentDidLoad() {
+    if (!this.uswds) {
+      return
+    }
+    // add uswds classes
+    const nodes = this.el.shadowRoot
+      .querySelectorAll('slot')
+    const headline = nodes[0];
+    const content = nodes[1];
+    
+    headline.classList.add('usa-summary-box__heading');
+    content.classList.add('usa-summary-box__text');
```

Comment:
I'm not sure if adding this class to a slot element does anything. Because when I try and fiddle with the CSS in the browser, nothing happens.

![Screenshot 2023-11-21 at 2 54 05 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/d5b9062a-2eb8-46af-98fd-3f748686488e)

I wonder if it needs to be on an actual element like `div` before it applies to child elements. 🤔

Reply:
Yeah, I'm conflicted. I don't see the point of these classes when they're not applying styles. I'm ok with leaving them since this is sorta its own component, and we can more strictly enforce a structure if we want to make an actual `summary-box` component

Reply:
Does it work if we turn off the component's shadow dom and some how programmatically apply the classes to an actual element (not a `slot`)? The purpose of having a USWDS v3 variation is to use their styling.

Reply:
We can at least add the `usa-summary-box__text` class though, right?

```
        <Host>
          <div class="usa-summary-box" role="region">
            <div class="usa-summary-box__body">
              <slot name="headline"></slot>
               <div class="usa-summary-box__text"> <------
                 <slot />
               </div>
            </div>
          </div>
        </Host>
```

Reply:
That class is already being added to the slot element dynamically.